### PR TITLE
SPM-1149: remove manifest file generation

### PR DESCRIPTION
**Pull request by MichaelMraka**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [x] Input Validation
- [x] Output Encoding
- [x] Authentication and Password Management
- [x] Session Management
- [x] Access Control
- [x] Cryptographic Practices
- [x] Error Handling and Logging
- [x] Data Protection
- [x] Communication Security
- [x] System Configuration
- [x] Database Security
- [x] File Management
- [x] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: deploy/clowdapp.yaml [hunk 1]
```diff
@@ -22,21 +22,6 @@ objects:
           enabled: true
       podSpec:
         image: ${IMAGE}:${IMAGE_TAG_MANAGER}
-        initContainers:
-          - command:
-            - /manifest/push.sh
-            - RedHatInsights/manifests
-            - ${MANIFEST_PUSH_BRANCH}
```
Comment (Josca) [inline deploy/clowdapp.yaml:29]: Please remove MANIFEST_PUSH_BRANCH from all places it is used.

Comment (Josca) [reply deploy/clowdapp.yaml:29]: We can also remove `github-patchman-bot` secret as it's used just to push manifest to github repo.

**Review by Josca — CHANGES_REQUESTED** [review]
Good, just couple more things to remove mentioned.

File: deploy/clowdapp.yaml (hunk 1 above)
Comment (MichaelMraka) [reply deploy/clowdapp.yaml:29]: removed `MANIFEST_PUSH_BRANCH` and `github-patchman-bot`

**Review by Josca — APPROVED** [review]
Thanks. Good to see sometimes we can also remove something not only adding things and making it more complicated :wink:

**Pull request merged**